app/views/api/v1/planning_applications/_show.json.jbuilder

```diff
@@ -10,4 +10,6 @@ json.local_authority planning_application.local_authority.name
 json.created_at planning_application.created_at.iso8601
 json.view_documents planning_application.view_documents
 json.uprn planning_application.property.uprn
+json.code planning_application.property.code
+json.type planning_application.property.type
```

Comment:
Since we're adding more fields related to a property, how about moving them on a property hash?

So the following:
```ruby
json.property do
  json.uprn planning_application.property.uprn
  json.code planning_application.property.code
  json.type planning_application.property.type
end
```
will be represented as
```json
"property": {
  "uprn": "XXXXXXXXXXX",
  "code": "CR08",
  "type": "Commercial, Retail, Shops and Showrooms"
}
```

Reply:
:nod: